pyproject.toml

```diff
@@ -4,3 +4,23 @@ build-backend = "setuptools.build_meta"
 
 [tool.pytest.ini_options]
 # addopts = -n auto
+
+[tool.tox]
+legacy_tox_ini = """
```

Comment:
Can we put  this giant string into `tox.ini` before tox gains proper support for pyproject.toml?